This is a divergence from upstream, where _DRM_GEM is removed.
Without this change x11/kernel will not build

Not suitable for upstream

--- a/include/drm/drm.h	2019-03-15 11:54:31.503726032 -0700
+++ b/include/drm/drm.h	2019-03-15 11:55:07.465384313 -0700
@@ -215,7 +215,8 @@
 	_DRM_SHM = 2,		  /**< shared, cached */
 	_DRM_AGP = 3,		  /**< AGP/GART */
 	_DRM_SCATTER_GATHER = 4,  /**< Scatter/gather memory for PCI DMA */
-	_DRM_CONSISTENT = 5	  /**< Consistent memory for PCI DMA */
+	_DRM_CONSISTENT = 5,	  /**< Consistent memory for PCI DMA */
+	_DRM_GEM = 6		  /**< GEM object */
 };
 
 /**
